### ISSUE TITLE
Attempting to click on a Worker from the Worker Pools page results in an incorrect URL

### ISSUE THREAD

**Issue by donnybell** [OctopusDeploy/Issues#8587]
### Version

2024.1.8396

### Latest Version

None

### What happened?

Attempting to click on a Worker from the Worker Pools page results in an incorrect URL.

Incorrect URL example:
`https://YOUR_OCTOPUS_URL/app#/Spaces-1/infrastructure/machines/Workers-21/connection`

Expected URL:
`https://YOUR_OCTOPUS_URL/app#/Spaces-1/infrastructure/workers/Workers-21/settings`

### Reproduction

1. Navigate to https://YOUR_OCTOPUS_URL/app#/Spaces-XX/infrastructure/workerpools
2. Expand a Worker Pool
3. Click on a Worker

### Error and Stacktrace

```shell
Resource is not found or it doesn't exist in the current space context. Please contact your administrator for more information.
```

### Workaround

Use the `Workers` page instead of the `Worker Pools` page. You can still filter by Worker Pool by selecting `Show Advanced Filters` option at the top of the page.

**Comment by donnybell** [comment]
This issue appears to be fixed in Octopus 2024.1.8619+ versions

**Comment by briggs-octo** [comment]
Cross-linking with issue #8666
